query/query.go

```diff
@@ -174,14 +198,29 @@ type Function struct {
 // query and the response. Once generated, this can then be encoded to other
 // client convenient formats, like GraphQL / JSON.
 type SubGraph struct {
```

Comment:
`query` and `subgraph` are interchangeably used. Not sure whether it makes sense to keep the distinction (or there is one)